### ISSUE TITLE
Feature request: Allow change deck during search

### ISSUE THREAD

**Issue by hamnad2** [fonol/anki-search-inside-add-card#257]
Hi, It would be nice to change deck for the searched notes in the pdf reader, this way I don't have to create cards that I already have in the database.

**Comment by fonol** [comment]
Hi, 
you mean for the tooltip (the one that appears when selecting text in a PDF)? Selecting the decks it searches in could be useful, although I admit I am happy enough with just searching in everything by default.

**Comment by hamnad2** [comment]
Yes exactly, I would love to change decks because I have big premade decks and I want to classify them into subdecks.
